en/release_note/release_note_latest_ver.rst

```diff
@@ -26,56 +28,69 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
 *   To upgrade drivers are highly recommended.
 
-For more details on changes, see the :ref:`11_2_changes`. Users of previous versions should check the :ref:`11_2_changes` and :ref:`11_2_new_cautions` sections.
+For more details on changes, see the :ref:`11_3_changes`. Users of previous versions should check the :ref:`11_3_changes` and :ref:`11_3_new_cautions` sections.
 
-.. _11_2_changes:
+.. _11_3_changes:
 
-11.2 Changes
+11.3 Changes
 ============
 
-Please refer to `change logs of CUBRID 11.2 <https://github.com/CUBRID/cubrid/releases/tag/v11.2>`_.
+Please refer to `change logs of CUBRID 11.3 <https://github.com/CUBRID/cubrid/releases/tag/v11.3>`_.
 
 
 Cautions
 ========
 
-.. _11_2_new_cautions:
+.. _11_3_new_cautions:
 
 New Cautions
 ------------
 
-The database volume of CUBRID 11.2 is not compatible with that of CUBRID 11.1 and earlier versions.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The query optimizer's statistical information is not automatically updated when DDL statement is executed, the user must manually execute UPDATE STATISTICS statement to update the statistical information. (see :doc:`/sql/tuning`)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When changing the column type of the AUTO_INCREMENT property in the ALTER TABLE statement, it cannot be changed to a type that cannot be used as the AUTO_INCREMENT property. When changing the type of a column where the default value is specified, an error occurs if type conversion of the specified default value to the changed type is not possible. (see :ref:`change-column`)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Unable to get Connection object through cubrid.jdbc.driver.CUBRIDDriver.getDefaultConnection() from server-side JDBC. Instead, you should use DriverManager.getConnection("jdbc\:default\:connection\:"). (see :ref:`jsp-server-side-jdbc-connection`)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If an invalid index name or table name is specified in the USE INDEX (USING INDEX) statement, it is ignored without processing an error. However, if the parameter error_log_warning is set to yes, it is left in the log file.
```

Comment:
```suggestion
If an invalid index name or table name is used in a USE INDEX (USING INDEX) statement, it will be ignored rather than occuring an error. (To leave it in the log file, set the error_log warning parameter to yes.)
```

Reply:
Reflected it